scripts/configure-installer-tests-cluster.sh

```diff
@@ -51,12 +51,16 @@ fi
 
 ## Create a new namesapce which will be used for OperatorHub checks
 oc new-project $CI_OPERATOR_HUB_PROJECT
-## Let developer user have access to the project
-oc adm policy add-role-to-user edit developer
-
 sh $SETUP_OPERATORS
 # OperatorHub setup complete
 
+## Create two namespace for authenticated registry e2e image test
+oc new-project openjdk-11-rhel8
+oc new-project nodejs-12-rhel7
+
+## Let developer user have access to the project
+oc adm policy add-role-to-user edit developer
```

Comment:
I think that this role is applied only to the current project. In this case, it will be only in `nodejs-12-rhel7` and not in any other namespaces.

Reply:
@amitkrout Lets iterate with on

```
oc new-project <project-name>
oc adm policy add-role-to-user edit developer
oc get secret pull-secret -n openshift-config -o yaml | sed 's/openshift-config/<project-name>/g' | oc apply -f -
```
with namespace.

Reply:
I did not know that. Thought that the policy will be applied to all the projects in the developer user.

Reply:
Done